client/src/components/fortune.vue

```diff
@@ -1,64 +1,77 @@
 <template>
-    <div class="container">
-      <div class="row">
-        <div class="col-sm-10">
-          <h1>
-              <img src="../assets/fortune-cookie.png" alt="fortune cookie" height="42" width="42">
-              Fortunes
-          </h1>
-          <hr><br><br>
-          <button type="button" class="btn btn-success btn-sm" v-b-modal.fortune-modal>Add Fortune</button>
-          <br><br>
-          <table class="table table-hover">
-            <thead>
-              <tr>
-                <th scope="col">Fortune</th>
-                <th scope="col">Author</th>
-                <th scope="col">Reviewed?</th>
-                <th></th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr v-for="(fortune, index) in fortunes" :key="index">
-                <td>{{fortune.fortune}}</td>
-                <td>{{fortune.author}}</td>
-                <td>
-                  <span v-if="fortune.approved">Yes</span>
-                  <span v-else>No</span>
-                </td>
-                <td>
-                  <button type="button"
-                          class="btn btn-info btn-sm"
-                          @click="editFortune(fortune)">Edit</button>
-                  <button type="button" class="btn btn-danger btn-sm">Delete</button>
-                </td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-10">
+        <h1>
+          <img src="../assets/fortune-cookie.png" alt="fortune cookie" height="42" width="42">
+          Fortunes
+        </h1>
+        <hr>
+        <br>
```

Comment:
Why so many br tags? If this is to make things spaced and look nice that should be handled in the css.

Reply:
This is what it looks like with the br tags:
![image](https://user-images.githubusercontent.com/1649836/55042242-e5a2b580-4fed-11e9-81de-ede83845b53b.png)
and without:
![image](https://user-images.githubusercontent.com/1649836/55042269-fd7a3980-4fed-11e9-858b-7ed8f497c375.png)

Reply:
I can probs just style the add fortune button to have more padding? what do you think